osu.Game/Screens/SelectV2/BeatmapLeaderboardWedge.cs

```diff
@@ -384,10 +381,18 @@ protected override void UpdateAfterChildren()
         {
             base.UpdateAfterChildren();
 
+            // Bottom padding ensures the last entry's full width is displayed (ie it is fully on screen after shear is considered).
+            // The padding is computed such that the leaderboard can be scrolled until the score is in the fifth position from the top (which is fully visible on screen).
+            // On high UI scales, the fifth position may be off-screen. A minimum sane value is set in place to handle this case (ensuring the last score sits above the footer by reasonable gap).
+            scoresContainer.Padding = scoresContainer.Padding with
+            {
+                Bottom = Math.Max(BeatmapLeaderboardScore.HEIGHT + 5, scoresScroll.DisplayableContent - BeatmapLeaderboardScore.HEIGHT * 5)
```

Comment:
Why is the `+ 5` only applied to the "minimum" portion? wouldn't it make more sense to apply it for both?

Reply:
At minimum, the padding should be HEIGHT + 5, otherwise it should be the other segment (i.e. more than HEIGHT + 5). Not sure I see why + 5 should be applied to the other side.